cpp-package/src/inc/data_filter.h

```diff
@@ -7,11 +7,31 @@
 #include "brainflow_exception.h"
 #include "data_handler.h"
 
+enum class LogLevels : int
+{
+    LEVEL_TRACE = 0,    /// TRACE
+    LEVEL_DEBUG = 1,    /// DEBUG
+    LEVEL_INFO = 2,     /// INFO
+    LEVEL_WARN = 3,     /// WARN
+    LEVEL_ERROR = 4,    /// ERROR
+    LEVEL_CRITICAL = 5, /// CRITICAL
+    LEVEL_OFF = 6       // OFF
+};
 
 /// DataFilter class to perform signal processing
 class DataFilter
 {
 public:
+    static void set_log_level (int log_level);
```

Comment:
I think it can be private and we can keep only enable\disable\enable_dev logs in user API